firebase-rules-coverage/package.json

```diff
@@ -28,17 +28,18 @@
     "release": "yarn clean && yarn compile && cp package.json ./build && cp README.md ./build && cp CHANGELOG.md ./build && yarn publish ./build"
   },
   "dependencies": {
-    "meow": "^12.0.1",
-    "source-map": "^0.7.3"
+    "meow": "^13.2.0",
+    "source-map": "^0.7.4"
   },
   "devDependencies": {
-    "@jest/globals": "^29.6.1",
-    "@types/jest": "^29.5.2",
-    "@types/node": "^15.6.1",
-    "gts": "^3.1.0",
-    "jest": "^29.6.1",
-    "ts-jest": "^29.1.1",
-    "ts-node": "^10.0.0",
-    "typescript": "^5.1.6"
-  }
-}
+    "@jest/globals": "^29.7.0",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^15.14.9",
+    "gts": "^5.3.0",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.1.4",
+    "ts-node": "^10.9.2",
+    "typescript": "~5.1.6"
```

Comment:
Would it make sense to bump the typescript version? `5.4.5` is the latest

Reply:
I wanted, but then all tests were crushed. I tried to figure out which dependencies produced it and via empirical method found out that only with ts version "5.1.x" tests are running well.